### PR TITLE
improved logging

### PR DESCRIPTION
**Pull request by sebader**
- Uses proper Serilog configuration
- Changes many LogInfo() to LogDebug()
- Remove unused request tracing by ASP.NETcore

### CODE REVIEW

**Comment by sebader** [comment]
Deployed a running version and had a locust run. Traces now only contain our own log entries
https://ms.portal.azure.com#@72f988bf-86f1-41af-91ab-2d7cd011db47/blade/Microsoft_Azure_Monitoring_Logs/LogsBlade/resourceId/%2Fsubscriptions%2Fcb8d2bb0-ed2c-44e5-a01b-cde33c0320a4%2FresourceGroups%2Fafe2ee3c2-monitoring-rg%2Fproviders%2Fmicrosoft.insights%2Fcomponents%2Fafe2ee3c2-northeuro-appi/source/LogsBlade.AnalyticsShareLinkToQuery/q/H4sIAAAAAAAAAyspSkxOLQYAoAfuugYAAAA%253D/timespan/P1D

@msimecek  would be great if you can review and give feedback if these traces are ok or should we move even more to debug()?

File: src/app/AlwaysOn.BackgroundProcessor/Services/EventHubProcessorService.cs [hunk 1]
```diff
@@ -251,8 +251,7 @@ private Task PartitionInitializingHandler(PartitionInitializingEventArgs eventAr
             try
             {
                 ownedPartitions.TryAdd(eventArgs.PartitionId, DateTime.UtcNow);
-                _logger.LogInformation("Initialized partition: {partitionId}", eventArgs.PartitionId);
-                _logger.LogInformation("Now owning {numberOfOwnedPartitions} partition(s)", ownedPartitions.Count);
+                _logger.LogInformation("Initialized partition: {partitionId} Now owning {numberOfOwnedPartitions} partition(s)", eventArgs.PartitionId, ownedPartitions.Count);
```
Comment (msimecek) [inline src/app/AlwaysOn.BackgroundProcessor/Services/EventHubProcessorService.cs:254]: 👍

Comment (sebader) [reply src/app/AlwaysOn.BackgroundProcessor/Services/EventHubProcessorService.cs:254]: I'm not sure whether I can resolve this conversation :D

File: src/app/AlwaysOn.CatalogService/Program.cs [hunk 2]
```diff
@@ -41,20 +41,17 @@ public static int Main(string[] args)
 
                 var builtConfig = config.Build();
                 Log.Logger = new LoggerConfiguration()
-                                    .MinimumLevel.Debug()
-                                    .MinimumLevel.Override("Microsoft", LogEventLevel.Information)
+                                    .ReadFrom.Configuration(builtConfig)
                                     .Enrich.FromLogContext()
-                                    .Filter.ByExcluding(c => c.Properties.Any(p => p.Value.ToString().ToLower().Contains("health/liveness"))) // Exclude pod health probe from logging
```
Comment (msimecek) [inline src/app/AlwaysOn.CatalogService/Program.cs:47]: No filtering anymore, because the log level does it?

**Review by msimecek — APPROVED** [review]
LGTM

File: src/app/AlwaysOn.CatalogService/Program.cs (hunk 2 above)
Comment (sebader) [reply src/app/AlwaysOn.CatalogService/Program.cs:47]: yes exactly, this is not needed anymore

**Pull request merged**